libs/shared/types/src/lib/types.ts

```diff
@@ -1,3 +1,5 @@
+import { InputType } from '@aws-sdk/client-medialive';
```

Comment:
We don't want to couple our shared types with the medialive library, we should redefine InputType ourselves here in this file.